### PR TITLE
[FEAT] - Creates env vars to handle our subgraph API endpoints

### PR DESCRIPTION
**Pull request by ElRodrigote**
## Context
With this change we will be able to update our subgraph API endpoint URLs for the three supported chains, this way we will be able not to consume API credits for our development.

## Description
* Creates env vars to handle our subgraph API endpoints

### CODE REVIEW

File: packages/sdk/src/vaults/constants.ts [hunk 1]
```diff
@@ -56,13 +56,26 @@ export const COW_SETTLEMENT_ADDRESS_LIST: Record<ChainId, string> = {
   [ChainId.ARBITRUM]: COW_SETTLEMENT_ADDRESS,
 };
 
-const STACKLY_SUBGRAPH_API_KEY =
+const API_BASE_URL = "https://gateway-arbitrum.network.thegraph.com/api";
+const SUBGRAPH_API_KEY =
   process.env.STACKLY_SUBGRAPH_API_KEY ?? "e7b7ff845e506590498946cd6bf83bf6";
```
Comment (berteotti) [inline packages/sdk/src/vaults/constants.ts:61]:
```suggestion
  process.env.STACKLY_SUBGRAPH_API_KEY ?? "";
```

Let's remove this key, we need to forcibly add it to env

File: packages/sdk/src/vaults/constants.ts [hunk 2]
```diff
@@ -56,13 +56,26 @@ export const COW_SETTLEMENT_ADDRESS_LIST: Record<ChainId, string> = {
...
 
+export const ETHEREUM_API_ENDPOINT_URL =
```
Comment (berteotti) [inline packages/sdk/src/vaults/constants.ts:63]: Let's call it `ETHEREUM_SUBGRAPH_ENDPOINT_URL`, same for Gnosis and Arbitrum

File: packages/sdk/src/vaults/constants.ts [hunk 3]
```diff
@@ -56,13 +56,26 @@ export const COW_SETTLEMENT_ADDRESS_LIST: Record<ChainId, string> = {
...
+  process.env.ETHEREUM_SUBRAPH_API_URL ??
```
Comment (berteotti) [inline packages/sdk/src/vaults/constants.ts:64]: typo, same for the others

```suggestion
  process.env.ETHEREUM_SUBGRAPH_API_URL ??
```

**Review by berteotti — COMMENTED** [review]
Looking good, just small improvements

**Review by berteotti — APPROVED** [review]

File: packages/sdk/src/vaults/constants.ts [hunk 4]
```diff
@@ -56,13 +56,25 @@ export const COW_SETTLEMENT_ADDRESS_LIST: Record<ChainId, string> = {
   [ChainId.ARBITRUM]: COW_SETTLEMENT_ADDRESS,
 };
 
-const STACKLY_SUBGRAPH_API_KEY =
-  process.env.STACKLY_SUBGRAPH_API_KEY ?? "e7b7ff845e506590498946cd6bf83bf6";
+const API_BASE_URL = "https://gateway-arbitrum.network.thegraph.com/api";
+const SUBGRAPH_API_KEY = process.env.STACKLY_SUBGRAPH_API_KEY ?? "";
```
Comment (Diogomartf) [inline packages/sdk/src/vaults/constants.ts:60]: if don't set the `SUBGRAPH_API_KEY` key wouldn't be better to break?

**Review by Diogomartf — APPROVED** [review]

**Pull request merged**